gradlew

```diff
@@ -1,7 +1,7 @@
-#!/usr/bin/env sh
+#!/bin/sh
 
 #
-# Copyright 2015 the original author or authors.
+# Copyright © 2015-2021 the original authors.
```

Comment:
2022 😄

Reply:
This is from the Gradle wrapper. 7.3.1 was released in 2021.